<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_listar_clientes</class>
 <widget class="QDialog" name="tela_listar_clientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selecionar Clientes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/img/img/logo_sex.png</normaloff>:/img/img/logo_sex.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="gb_buscar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>780</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Buscar por:</string>
   </property>
   <widget class="QLabel" name="lb_codigo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CÃ³digo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>23</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_rg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>RG:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>23</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cpf">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>50</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_rg">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>00000000000000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="cursorPosition">
     <number>20</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nome">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>23</y>
      <width>430</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>410</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cpf">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>50</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>000.000.000-00; </string>
    </property>
    <property name="text">
     <string>..-</string>
    </property>
    <property name="maxLength">
     <number>14</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_telefone">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>23</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Telefone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_email">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_telefone">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>23</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>(00)00000-0000; </string>
    </property>
    <property name="text">
     <string>()-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>50</y>
      <width>390</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>77</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/busar_clientes.png</normaloff>:/img/img/busar_clientes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>77</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/1345902747_button_cancel.png</normaloff>:/img/img/1345902747_button_cancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpar">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>77</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/limpar.png</normaloff>:/img/img/limpar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_clientes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>780</width>
     <height>240</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Clientes</string>
   </property>
   <widget class="QTableView" name="tv_clientes">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>15</y>
      <width>766</width>
      <height>218</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_codigo</tabstop>
  <tabstop>le_nome</tabstop>
  <tabstop>le_telefone</tabstop>
  <tabstop>le_rg</tabstop>
  <tabstop>le_cpf</tabstop>
  <tabstop>le_email</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>btn_cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_codigo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_nome</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_telefone</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>711</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_rg</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_cpf</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_limpar</sender>
   <signal>clicked()</signal>
   <receiver>le_email</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
